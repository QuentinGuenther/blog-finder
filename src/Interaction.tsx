import * as React from 'react';

export interface InteractionProps {
  comments: number;
  votes: number;
}

export function Interaction (props: InteractionProps) {
  return (
    <div>
      <div className="comments">{props.comments}</div>
      <div className="vote">{props.votes}</div>
    </div>
  );
}
