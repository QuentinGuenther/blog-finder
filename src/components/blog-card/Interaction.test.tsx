import * as React from 'react';
import renderer from 'react-test-renderer';
import { Interaction } from './Interaction';

describe('Interaction should match snapshot', () => {
  it('Interaction should match snapshot', () => {
    const tree = renderer
      .create(<Interaction comments={75} votes={105000} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});