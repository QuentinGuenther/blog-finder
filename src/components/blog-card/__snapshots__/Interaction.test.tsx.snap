// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interaction should match snapshot Interaction should match snapshot 1`] = `
<div>
  <div
    className="comments"
  >
    75
  </div>
  <div
    className="vote"
  >
    105000
  </div>
</div>
`;
